securethenews/requirements.in

```diff
@@ -6,7 +6,6 @@ django-filter>=2.1.0
 -e git+https://github.com/freedomofpress/django-logging.git@51f4fd4b02fafc3d6d9cddb5658247497404df7e#egg=django-logging-json
 django-mailgun
 django-storages[boto3, google]>=1.7 # aws s3, google storage asset management
-django-webpack-loader
```

Comment:
We do indeed want `django-webpack-loader`! That's used to collect the static assets, as of #181.

Reply:
Not sure why this got removed. I have added it back in my working branch.